src/FirstSteps.Tests/RPG/Adventures/MineTests.cs

```diff
@@ -7,6 +7,35 @@ namespace FirstSteps.Tests.RPG.Adventures
 {
     public class MineTests
```

Comment:
General comment. Try to write unit tests with arrange/act/assert comments above each section. It will help you to understand the steps. 
```csharp
public void Test()
{
    //arrange 
    ...

    //act
    ...

    //assert
    ...
}
```